src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1766,3 +1760,25 @@ def list_locations(cmd, sku, linux_workers_enabled=None):
     client = web_client_factory(cmd.cli_ctx)
     full_sku = get_sku_name(sku)
     return client.list_geo_regions(full_sku, linux_workers_enabled)
+
+
+def _check_zip_deployment_status(deployment_status_url, authorization):
+    import requests
+    import time
+    num_trials = 1
+    while num_trials < 200:
+        time.sleep(45)
```

Comment:
`45` second is rather long which would make people feel your command is hanging. I suggest you out a warning to tell people you are waiting 45 seconds to re-poll

Reply:
how about i decrease the wait to 15 seconds?

Reply:
Let me make these updates & commit, that you can then merge